### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Oscar-Jin**
add section "code of conduct"

### CODE REVIEW

**Comment by Oscar-Jin** [comment]
Hi I have made some updates on "CONTRIBUTING.md".

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Contributing to Learn Open Source
 
-Guideline for contribution
-
 Thank you for investing your time in contributing to our project! 
+
+## Code of Conduct
+
+We take our open source community seriously and hold ourselves and other contributors to high standards of communication.
```
Comment (owfdr) [inline CONTRIBUTING.md:7]: it might be better to keep it in a single line :)

**Review by owfdr — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (owfdr) [reply CONTRIBUTING.md:7]: others are seem to be fine to me!

Comment (Oscar-Jin) [reply CONTRIBUTING.md:7]: okay will do that!

Comment (owfdr) [reply CONTRIBUTING.md:7]: thanks!

**Comment by Oscar-Jin** [comment]
Hi I have finished my refactor

**Review by owfdr — APPROVED** [review]
Perfect!

**Comment by owfdr** [comment]
Thanks for the help!

**Pull request merged**